CryptKeeperZK/crypt-keeper-extension
# feat: enable reset password feature

## Explanation

This PR enables recover access and password reset feature for users.

## More Information

Related to #381
Blocked by #614 

## Screenshots/Screencaps

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/f7a5b17f-befe-4527-9315-7aa65fa2ec80)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/8720ee30-4251-4c93-86c1-bc51c7f7f31b)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/2e009635-ae09-4fee-8375-dd87d3a3a1e9)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/5323c947-3ccd-49b1-a8d3-2a4b81ccf7c6)


## Manual Testing Steps

1. Go to login page
2. Click "Forgot password?"
3. Complete recover form with mnemonic phrase
4. Set new password
5. Open Activity tab and check if event is tracked
6. Lock the extension
7. Login with new password

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
